src/conn/conn_dhandle.c

```diff
@@ -48,6 +51,9 @@ __conn_dhandle_config_set(WT_SESSION_IMPL *session)
             ret = __wt_set_return(session, ENOENT);
         WT_RET(ret);
     }
+    cfg[0] = metaconf;
+    cfg[1] = NULL;
+    cfg[2] = NULL;
```

Comment:
We don't need this initialization in the case of `WT_DHANDLE_TYPE_TABLE`.

Reply:
Moved.